### DISCUSSION TITLE
Edit labels and populate minibuffer with existing selection

### DISCUSSION THREAD

**Discussion by sergiodj** [magit/forge#731] [Support] [answered]
Hi,

When I try to edit GitLab labels (but I'm assuming this doesn't only affect GitLab) by hitting `C-c C-e` inside an issie or by using `N m s -l` when viewing a list of issues, forge offers in the minibuffer the available labels that can be applied.  However, there is no way (or at least I could not find one) to populate the prompt with the *current* labels that are already applied to the issue.  This is cumbersome especially when dealing with issues that have more than 2 or 3 labels attached.  Suppose that I want to remove a single label from an issue that has e.g. 6 labels attaced to it.  I'd need to manually type the 5 remaining labels.

Is there a way to have the list of active labels show up in the prompt?  I tried `M-n`, `M-p` and many other key combinations to no avail.

Thanks.

**Comment by tarsius** [comment, marked as answer]
Thanks for the report. Fixed in a69b87d8fe62de16e620297b3212cab78b6937d2.
